tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py{27,35},coverage,pylint
 [testenv]
 deps =
     coverage
+    Django>=1.8,<1.9
```

Comment:
This should probably be updated to use Django 1.11, since 1.8 isn't supported anymore.

Reply:
Will do.